codegen/lib/graphql_java_gen.rb

```diff
@@ -37,7 +38,7 @@ class << self
     private
 
     def erb_for(template_filename)
-      erb = ERB.new(File.read(template_filename))
+      erb = ERB.new(File.read(template_filename), 0, '-')
```

Comment:
Why are you also changing the `safe_level` argument from `nil` to non-nil.

> If safe_level is set to a non-nil value, ERB code will be run in a separate thread with $SAFE set to the provided level.

Seems like you should just use `nil` for that argument.